main/src/org/destinationsol/SolApplication.java

```diff
@@ -250,7 +248,8 @@ public OggSoundManager getSoundManager() {
         return soundManager;
     }
 
-    public void paused() {
+    // TODO: Why do we even have this method? Look into it's removal.
+    public void pauseApplication() {
```

Comment:
The function for application losing focus is pause(), the paused() (now pauseApplication() for clarity) is an entirely different function that is never used - which is why inspect for removal, with the only thing being maybe saving the logic inside it somewhere.

Also, fixed the typo :D Appreciate the nitpick :)
(Idk why I somtimes use it's instead of its without thinking)